.github/actions/create-jira-release-ticket/README.md

```diff
@@ -0,0 +1,101 @@
+# Create Jira Release Ticket Action
```

Comment:
I'm slightly confused about what's the relation between this repo and https://github.com/SonarSource/gh-action_release

I've recently raised a point to Eng. Xp Squad about difficulty of analyzer releasing and the time wasted on those in general and they plan to take a look into that eventually. Was this effort coordinated with them?